pupa/cli/commands/update.py

```diff
@@ -232,7 +253,17 @@ def check_session_list(self, juris):
             raise CommandError('session(s) unaccounted for: %s' % ', '.join(unaccounted_sessions))
 
     def handle(self, args, other):
-        juris = self.get_jurisdiction(args.module)
+        juris, module = self.get_jurisdiction(args.module)
+        overrides = {}
+        overrides.update(getattr(module, 'settings', {}))
```

Comment:
OK, just so I'm sure I'm processing this logic correctly, we're taking {module}.settings and overriding anything from {pupa}.settings?